.github/workflows/pull-requests.yaml

```
@@ -346,8 +357,8 @@ jobs:
 
   cleanup:
     name: Tear down environment
-    runs-on: [self-hosted]
-    needs: [collect_debug_information]
+    runs-on: ${{ needs.capture_runner.outputs.runner_name }}
+    needs: [capture_runner, collect_debug_information]
     if: ${{ always() && needs.test_apps.result == 'success' }}
```

actionlint (1.7.9)

property "test_apps" is not defined in object type {capture_runner: {outputs: {runner_name: string}; result: string}; collect_debug_information: {outputs: {}; result: string}}

(expression)